src/main/java/com/fvv/bookstore/view/BookViewImpl.java

```diff
@@ -68,7 +67,7 @@ public void addBook() {
 			e.printStackTrace();
```

Comment:
you can group this exception with the ControllerException since the implementation is the same. Ex catch (BookValidationException | ControllerException  e).....

Reply:
ok